src/commands/goto.ts

```diff
@@ -50,6 +59,70 @@ function executeGoto(gotoType: number, editor: vscode.TextEditor, count: number,
   }
 }
 
+function executeGotoLine(editor: vscode.TextEditor, count: number, extend: boolean, position: string = 'default') {
+  const positions: { [index: string]: {(x: vscode.Selection): number} } = {
```

Comment:
Removing that explicit type annotation will make type-checking slightly better (because the index will need to be among `'first' | 'end' | 'start' | 'default'`).